content-src/asrouter/templates/SimpleBelowSearchSnippet/SimpleBelowSearchSnippet.schema.json

```diff
@@ -39,6 +45,34 @@
       "description": "Tooltip text used for dismiss button.",
       "default": "Remove this"
     },
+    "button_action": {
+      "type": "string",
+      "description": "The type of action the button should trigger."
+    },
+    "button_url": {
+      "allOf": [
+        {"$ref": "#/definitions/link_url"},
+        {"description": "A url, button_label links to this"}
+      ]
+    },
+    "button_action_args": {
+      "type": "string",
+      "description": "Additional parameters for button action, example which specific menu the button should open"
+    },
+    "button_label": {
+      "allOf": [
+        {"$ref": "#/definitions/plainText"},
```

Comment:
You need to add `plainText` to the `definitions` object at the top so this will work

Reply:
You might want to also add dependencies at the bottom:
```
"dependencies": {
    "button_action": ["button_label"],
    "button_url": ["button_label"],
    "button_color": ["button_label"],
    "button_background_color": ["button_label"],
  }
```